tests/test_config.py

```diff
@@ -1755,6 +1758,83 @@ def test_copy(self):
         self.assertNotEqual(configA.run_config_search_mode,
                             configB.run_config_search_mode)
 
+    def test_multi_model_search_mode(self):
+        """
+        Test that multi-model is only run in quick
+        """
+        args = [
+            'model-analzyer', 'profile', '--model-repository', 'cli-repository',
+            '--profile-models', 'test_modelA,test_modelB',
+            '--run-config-profile-models-concurrently-enable'
+        ]
+
+        yaml_content = ''
+
+        # Tests the case where no search mode is specified (default is brute)
+        with self.assertRaises(TritonModelAnalyzerException):
+            self._evaluate_config(args, yaml_content, subcommand='profile')
+
+        args.append('--run-config-search-mode')
+        args.append('brute')
+
+        with self.assertRaises(TritonModelAnalyzerException):
+            self._evaluate_config(args, yaml_content, subcommand='profile')
```

Comment:
add the passing case (where it properly uses quick)

Reply:
Added